### PR TITLE
Unit 5: Automatic Speech Recognition

### PR DESCRIPTION
**Pull request by sanchit-gandhi**

### CODE REVIEW

**Review by MKhalusova — COMMENTED** [review]
Hi! 
I did another pass on Unit 05. There are a couple of nits, and some todos left, but probably the largest TODO item is updating the example to Sinhalese instead of Dhivehi, and adding a hands-on exercise. I can work on the hands-on exercise part.

File: chapters/en/chapter5/evaluation.mdx [hunk 1]
```diff
@@ -1 +1,310 @@
...
+from datasets import load_dataset
+
+common_voice_test = load_dataset("mozilla-foundation/common_voice_13_0", "dv", split="test")
+```
+
+Evaluating over an entire dataset can be done in much the same way as over a single example - all we have to do is **loop**
+over the predictions, rather than inferring just the first one:
+
+```python
+from tqdm import tqdm
```
Comment (MKhalusova) [inline chapters/en/chapter5/evaluation.mdx:260]:
```suggestion
from transformers.pipelines.pt_utils import KeyDataset
from tqdm import tqdm
```

File: chapters/en/chapter5/evaluation.mdx [hunk 2]
```diff
@@ -1 +1,310 @@
...
+
+all_predictions = []
+for prediction in tqdm(pipe(common_voice_test, max_new_tokens=256, generate_kwargs={"task": "transcribe"})):
```
Comment (MKhalusova) [inline chapters/en/chapter5/evaluation.mdx:263]:
```suggestion
for prediction in tqdm(pipe(KeyDataset(common_voice_test, "audio"), max_new_tokens=256, generate_kwargs={"task": "transcribe"})):
```

File: chapters/en/chapter5/evaluation.mdx [hunk 3]
```diff
@@ -1 +1,310 @@
...
+section on fine-tuning by evaluating the pre-trained Whisper model on the Common Voice 13 Dhivehi test set. We'll use
+the WER number we get as a _baseline_ for our fine-tuning run, or a target number that we'll try and beat 🥊
+
+First, we'll load the pre-trained Whisper model using the `pipeline()` class. This process will be extremely familiar by now!
+
+```python
+from transformers import pipeline
+import torch
+
+device = "cuda:0" if torch.cuda.is_available() else "cpu"
```
[the remark this answers is not in the document]
Comment (sanchit-gandhi) [reply chapters/en/chapter5/evaluation.mdx:244]: We can with `device_map="auto"` - but setting `device` is preferred for single GPU inference

**Pull request closed without merging**